src/test/java/org/sagebionetworks/bridge/config/PropertiesConfigTest.java

```diff
@@ -183,6 +183,26 @@ public void testOverwriteFromString() throws IOException, URISyntaxException {
         Path localPath = Paths.get(getClass().getClassLoader().getResource("conf/local.conf").toURI());
         System.setProperty(PropertiesConfig.ENV_KEY, "dev");
         Config config = new PropertiesConfig(TEST_CONF_FILE, localPath);
-        assertEquals("local.value.for.dev", config.get("example.property"));
+        assertEquals(config.get("example.property"), "local.value.for.dev");
```

Comment:
Ok.  I changed it so that the new tests conforms to existing pattern even though the existing patter is wrong.  The correct pattern is assertEquals(actual, expected).  I'll leave refactoring to the correct pattern in a separate change.